d3a_api_client/utils.py

```diff
@@ -369,3 +370,12 @@ def read_simulation_config_file(config_file_path):
 def get_sim_id_and_domain_names():
     return simulation_id_from_env(), domain_name_from_env(), websocket_domain_name_from_env()
 
+
+def validate_client_up_to_date(response):
+    if "Api-Version" in response.headers:
+        remote_version = response.headers.get("Api-Version")
+        from d3a_api_client import __version__
```

Comment:
Is it necessary to have this import here, or could it be moved above at module level?

Reply:
@Hassan754 upping this question in case you haven't seen it :)

Reply:
Yes we can safely remove the local import